tests/phpunit/E2E/Core/ErrorTest.php

```diff
@@ -97,7 +97,7 @@ public function testErrorChrome(string $url, string $errorType) {
       'Backdrop' => '/body class=\".*not-logged-in/',
```

Comment:
@totten this doesn't seem to work on Backdrop btw

Reply:
@seamuslee001 Quite right. Sent up a parallel PR for that: #23398